frontend/packages/topology/src/actions/provider.ts

```diff
@@ -37,3 +38,12 @@ export const useTopologyVisualConnectorActionProvider = (element: Edge) => {
     return [actions, !inFlight, undefined];
   }, [actions, inFlight]);
 };
+
+export const useVmTopologyActionsProvider = (element: Node) => {
+  const vm: VMKind = getResource(element);
```

Comment:
Kubevirt package is going to be moved out of console repo to a dynamic plugin. We can't really use VMKind as dependency here.

Reply:
alright, should I move it back to kubevirt?

Reply:
Can you use something generic here instead of VMKind? K8sResourceKind maybe

Reply:
thanks, I have used K8sResourceKind now